modules/fastcdr/2.1.3/MODULE.bazel

```diff
@@ -0,0 +1,18 @@
+module(
+    name = "fastcdr",
+    version = "2.1.3",
+    compatibility_level = 1,
+)
+
+bazel_dep(
```

Comment:
Since Python and C++ rules are still shipped with Bazel, these two deps aren't needed.

Reply:
remove unnecessary deps at https://github.com/bazelbuild/bazel-central-registry/pull/1345/commits/5f9045114168aa8ef27be366ae08977e562bf1bf